.changeset/nasty-ravens-live.md

```diff
@@ -0,0 +1,5 @@
+---
+"@livekit/rtc-node": patch
```

Comment:
```suggestion
"@livekit/rtc-node": minor
```

? cc @theomonnom your call

Reply:
yeah you're probably right. i changed that but am happy to change back